### PR TITLE
Feature/47 Bug Report

### PR DESCRIPTION
**Pull request by christianrd**
## What's new? ##

- [x] Formulario en el sitio web
- [x] Implementar Captcha
- [ ] Al mandarlo creamos un issue en github y  Le ponemos un tag de "User submitted
- [ ] Notificamos en un canal de slack

### Visual aids ###

If there are visual changes include them here. You can add a screenshot or a recording using a tool like [this one](https://recordit.co/) and include it in the Pull Request.

This is the markdown to include an image in the pull request:

```md
![myimageurl.gif](myimageurl.gif);
```

You can include a comparison table as follows:

| Before | After   |
|---|---|
| Insert images from before the fix | Here is the after |

### Issue: #47

### CODE REVIEW

**Comment by eatskolnikov** [comment]
Haz un rebase de tu branch antes de continuar plz. Hicimos unos cambios.

File: Web/Framework/Configurations/IocConfiguration.cs [hunk 1]
```diff
@@ -31,6 +32,7 @@ public static void Init(IConfiguration configuration, IServiceCollection service
             //services.AddSingleton<IJobsService, MockJobsService>();
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
             services.AddSingleton<ISlackService, SlackService>();
+            services.AddSingleton<IReCaptchaService, ReCaptchaService>();
```
Comment (erickconcepcion) [inline Web/Framework/Configurations/IocConfiguration.cs:35]: Por que singleton? Siempre se necesita una instancia arriba para que funcione? Por que no scoped?

Comment (christianrd) [reply Web/Framework/Configurations/IocConfiguration.cs:35]: Estaba pensando que se debería implementar el captcha en el wizard de crear vacante.

**Comment by codercampos** [comment]
Anything to push this forward?

**Pull request closed without merging**